### ISSUE TITLE
The babashka pod doesn't mark `.babashka-pod-254693.port` files for deletion

### ISSUE THREAD

**Issue by sheluchin** [epiccastle/spire#153]
I know that I should be using `bbssh` at this point, but I haven't had time to upgrade everything yet, so here is a report for 0.1.1...

Given the following `test.clj`:
```clojure
(ns test
  (:require [babashka.pods :as pods]))

(pods/load-pod 'epiccastle/spire "0.1.1")

(require '[pod.epiccastle.spire.module.shell :as shell])

(shell/shell {:cmd "echo 123"})
```

Running it produces `.babashka-pod-nnnnnn.port` files which remain after the process shuts down:
```shell
$ ll
total 4.0K
-rw-rw-r-- 1 alex 182 Nov 20 16:59 test.clj
$ bb test.clj
test.clj:8 (shell/shell {:cmd "echo 123"}) local
{:exit 0, :out "123\n", :err "", :out-lines ["123"], :result :ok}
$ ll
total 8.0K
-rw-rw-r-- 1 alex 182 Nov 20 16:59 test.clj
-rw-rw-r-- 1 alex   6 Nov 20 16:59 .babashka-pod-269082.port
$ bb test.clj
test.clj:8 (shell/shell {:cmd "echo 123"}) local
{:exit 0, :out "123\n", :err "", :out-lines ["123"], :result :ok}
$ bb test.clj
test.clj:8 (shell/shell {:cmd "echo 123"})
test.clj:8 (shell/shell {:cmd "echo 123"}) local["123"], :result :ok}
$ ll
total 16K
-rw-rw-r-- 1 alex 182 Nov 20 16:59 test.clj
-rw-rw-r-- 1 alex   6 Nov 20 16:59 .babashka-pod-269082.port
-rw-rw-r-- 1 alex   6 Nov 20 16:59 .babashka-pod-269516.port
-rw-rw-r-- 1 alex   6 Nov 20 16:59 .babashka-pod-269838.port
$ 
```

My understanding is that when the process shuts, the port files _should_ be deleted, but are not.

Thank you for your work on spire!

**Comment by retrogradeorbit** [comment]
`bbssh` is not intended to replace spire. Spire will eventually use bbssh internally. So this is a bug that needs to be fixed in the meantime.

**Comment by sheluchin** [comment]
Works correctly as of the last commit. Thanks, @retrogradeorbit!

**Comment by retrogradeorbit** [comment]
released in 0.1.2
